depmapomics/copynumbers.py

```diff
@@ -549,67 +552,48 @@ def maskGenes(
     overlap_rm = pd.read_csv(
         save_output + "mask_overlap_rm.bed",
         sep="\t",
-        names=["chrom", "start", "end", "gene_name"],
+        names=["chrom", "start", "end", "ensembl_gene_id"],
     )
 
     gene_dict = (
-        mybiomart[["Chromosome", "start", "end", "gene_name"]]
-        .set_index("gene_name")
+        mybiomart[["Chromosome", "start", "end", "ensembl_gene_id"]]
+        .set_index("ensembl_gene_id")
         .T.to_dict("list")
     )
 
-    gene2gene_entrez_dict = dict(zip(mybiomart["hgnc_symbol"], mybiomart["gene_name"]))
-
     to_rescue = open(rescue_list, "r").read().split("\n")
```

Comment:
While we're here: 1) wrap in `with open():` block, 2) use `.splitlines()` instead of `.split()`